### ISSUE TITLE
GURA02 should not warn here

### ISSUE THREAD

**Issue by JohanLarsson** [GuOrg/Gu.Roslyn.Asserts#116]
```cs
    internal static class SplitStringRefactoringTests
    {
        private static readonly CodeRefactoringProvider Refactoring = new SplitStringRefactoring();

        [Test]
        public static void StartingWithNewLine()
        {
            var before = @"
namespace RoslynSandbox
{
    public static class C
    {
        public static void Test()
        {
            var text = ""↓\na"";
        }
    }
}";

            var after = @"
namespace RoslynSandbox
{
    public static class C
    {
        public static void Test()
        {
            var text = ""\n"" +
                       ""a"";
        }
    }
}";
            RoslynAssert.Refactoring(Refactoring, before, after);
        }
```

![image](https://user-images.githubusercontent.com/1640096/61135464-f5bda280-a4c1-11e9-8eaf-042c4a0ea89c.png)

Two warnings.
